Makefile

```diff
@@ -19,6 +19,8 @@ jupyter_python_version_without_dot := $(shell echo $(jupyter_python_version) | s
 jupyter_notebook_image_repository := jupyter/scipy-notebook:x86_64-python-$(jupyter_python_version).3
 extract_python_package_version := $(shell echo $(project_version) | sed 's/-/./' | sed 's/-.*//')
 
+dev_username := vscode
```

Comment:
I know we all use vscode, but I would say that we want to keep the dev tools independent of an editor since this is an open source project.

Reply:
This actually doesn't have anything to do with VSCode itself, its just the name of the non-root user that is configured in the docker image for the `development-image-non-root` at build time. It just needs to match what is specified in line-189 of the development Dockerfile.

But you bring up a good point, I'll change the name to something more generic like `developer`.